src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HomePage {
+    private WebDriver driver;
+
+    private By searchLoc=By.xpath("//*[@id='jobsfilter-category']");
+    private By categoriesLoc=By.xpath("//option[text()='Web/Graphic design']");
```

Comment:
It's good practice to make changeable part- in this case the category name, a variable, so you can pass the category name from test method and you code will be more flexible and readable.